app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user
+
+  def create
```

Comment:
You could probably pull out the loading of q/a as commentable into its own private method and just call it here - you have some unnecessary repetition.

```
def create
  commentable = grab_commentable
  @comment = commentable.comments.build(comment_params.merge(commentator: current_user))
  if @comment.save
    render @comment
  else
    redirect_to question_path(@question), flash: {error: "Comment must be 1500 chars or less." }
  end
end

private
 
def grab_commentable
  if params[:answer_id]
    commentable = Answer.find_by(id: params[:answer_id])
  else
    commentable = Question.find_by(id: params[:question_id])
  end
end
```